tox.ini

```diff
@@ -27,15 +27,14 @@ deps = -rtests_utils/test-requirements.txt
 commands =
     python setup.py nosetests {posargs}
     bash -c "python setup.py lint | tee pylint.out || true"
-    bash -c "python setup.py pep8 | tee pep8.out || true"
-
+    bash -c "flake8 | tee flake8.out || true"
```

Comment:
I would keep the pep8 and add the flake8 check

Reply:
done, it should pass in travis